multicast/src/main/kotlin/com/dropbox/flow/multicast/ChannelManager.kt

```diff
@@ -275,20 +280,23 @@ internal class ChannelManager<T>(
          */
         private val channel: SendChannel<Message.Dispatch.Value<T>>,
         /**
-         * Tracking whether we've ever dispatched a value or an error to downstream
+         * Tracking whether this channel a piggyback only channel that can be closed without ever
+         * receiving a value or error.
          */
-        private var _receivedValue: Boolean = false
+        val piggybackOnly: Boolean = false
     ) {
-        val receivedValue
-            get() = _receivedValue
+        private var _stillRequiresDispatch: Boolean = !piggybackOnly
+
+        val stillRequiresDispatch
```

Comment:
rename suggestion: `awaitsDispatch` (feel free to ignore, still requires sounded a bit weird to me)